app/src/main/java/com/sdp/movemeet/Backend/FirebaseInteraction.java

```diff
@@ -87,4 +90,44 @@ public static void logoutIfUserNonNull(FirebaseAuth fAuth, Activity activity) {
         }
     }
 
+    public static void getImageFromFirebase(StorageReference imageRef, ImageView imageView, ProgressBar progressBar) {
```

Comment:
This is an interface for get and upload an image in Firebase ?

Reply:
Well, this is simply a refactored method implemented in the FirebaseInteraction class to avoid code duplicates (because this "getImageFromFirebase" function is also used in EditProfileActivity).